### DISCUSSION TITLE
Support for YT Premium Features (Higher Resolutions, Bitrates, ...)

### DISCUSSION THREAD

**Discussion by pschichtel** [FreeTubeApp/FreeTube#5309] [Q&A]
I really like the idea of FreeTube, but I'm also a Premium subscriber and like the higher resolutions and bitrates of videos. Is it possible for FreeTube to use my YT account to allow streaming in higher quality?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Using your YouTube account is against the goals of the FreeTube project as this is a privacy first application. FreeTube will only show all the content users see on YouTube logged out state.
